src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.cs

```diff
@@ -298,6 +298,11 @@ private static void BindProperty(PropertyInfo property, object instance, IConfig
                 return;
             }
 
+            if (config == null)
```

Comment:
Previously the entire block below was wrapped in a null check - https://github.com/dotnet/runtime/commit/35f043f414dbd234978f0eb694b47dfb42dd2031#diff-2e6f75e1d4f577e1c8c1b6a32d158ce7812a38ad8fcd3b4d762157b92c03e890L308-R315

Since this method already has multiple returns, I find it more readable to early-exit rather than count curly-brackets/indentation.

Reply:
sounds good to me.